broker/openshift-origin-broker.spec

```diff
@@ -79,6 +79,7 @@ Requires:      %{?scl:%scl_prefix}rubygem-metaclass
 Requires:      %{?scl:%scl_prefix}rubygem-mime-types
 Requires:      %{?scl:%scl_prefix}rubygem-moped
 Requires:      %{?scl:%scl_prefix}rubygem-multi_json
+Requires:      %{?scl:%scl_prefix}rubygem-net-ssh
```

Comment:
@sdodson ahh, I see. I'll update my PR with this additional change. 

Just to clarify; you want me to take it out of broker/openshift-origin-broker.spec and add it to controller/rubygem-openshift-origin-controller.spec ?